### PR TITLE
Documentation: Add colormap options to docs

### PR DESCRIPTION
**Pull request by gjmooney**

### CODE REVIEW

File: docs/layers/choropleth.rst [hunk 1]
```diff
@@ -88,6 +88,12 @@ The ``Choropleth`` layer is then created specifying on which key the colormap is
     )
 
 
+Other options for the colormap can be found by running ``linear`` in the notebook cell:
```
Comment (martinRenou) [inline docs/layers/choropleth.rst:91]: Maybe we can also point to the branca documentation there: https://python-visualization.github.io/branca

**Review by martinRenou — COMMENTED** [review]
Thank you! I just have a minor suggestion

File: docs/layers/choropleth.rst (hunk 1 above)
Comment (gjmooney) [reply docs/layers/choropleth.rst]: Good idea, I Added the link.

**Review by martinRenou — APPROVED** [review]
Thanks!

**Comment by martinRenou** [comment]
@gjmooney did you mean to close this PR? It seems we forgot to merge this prior to making the release

**Comment by gjmooney** [comment]
> @gjmooney did you mean to close this PR? It seems we forgot to merge this prior to making the release

Woops, I though we merged it already.

**Comment by martinRenou** [comment]
Are you able to press the "reopen" button?

**Comment by gjmooney** [comment]
Nope, should I open a new PR?

**Pull request closed without merging**